docs/source/configuration/entity-caching.mdx

```diff
@@ -0,0 +1,140 @@
+
+### Benefits of entity caching
+
+Compared to caching entire client responses, entity caching supports finer control over:
+- the time to live (TTL) of cached data
+- the amount of data being cached 
+
+When caching an entire client response, the router must store it with a shorter TTL because application data can change often. Real-time data needs more frequent updates.
+
+A client-response cache might not be shareable between users, because the application data might contain personal and private information. A client-response cache might also duplicate a lot of data between client responses.
+
+For example, consider the `Products` and `Inventory` subgraphs from the [Entities guide](/federation/entities):
+
+<CodeColumns>
+
+```graphql title="Products subgraph" disableCopy=true
+type Product @key(fields: "id") {
+  id: ID!
+  name: String!
+  price: Int
+}
+```
+
+```graphql title="Inventory subgraph" disableCopy=true
+type Product @key(fields: "id") {
+  id: ID!
+  inStock: Boolean!
+}
+```
+
+</CodeColumns>
+
+Assume the client for a shopping cart application requests the following for each product in the cart:
+- The product's name and price from the `Products` subgraph.
+- The product's availability in inventory from the `Inventory` subgraph.
+
+If caching the entire client response, it would require a short TTL because the cart data can change often and the real-time inventory has to be up to date. A client-response cache couldn't be shared between users, because each cart is personal.  A client-response cache might also duplicate data because the same products might appear in multiple carts.
+
+With entity caching enabled for this example, the router can:
+- Store each product's description and price separately with a long TTL.
+- Minimize the number of subgraph requests made for each client request, with some client requests fetching all product data from the cache and requiring no subgraph requests.
+- Share the product cache between all users.
+- Cache the cart per user, with a small amount of data.
+- Cache inventory data with a short TTL or not cache it at all.
+
+## Use entity caching
+
+Follow this guide to enable and configure entity caching in the Apollo Router.
+
+### Prerequisites
+
+To use entity caching in the Apollo Router, you must set up:
+
+- A Redis instance or cluster that your router instances can communicate with
+- A [GraphOS Enterprise plan](https://www.apollographql.com/pricing/) that [connects your router to GraphOS](./overview/#environment-variables).
+
+### Configure router for entity caching
+
+In `router.yaml`, configure `preview_entity_cache`:
+- Enable entity caching globally.
+- Configure Redis using the same conventions described in [distributed caching](./distributed-caching).
```

Comment:
Do we want users to read the entire distributed caching guide, or is it enough to point them to the Redis URL configuration section?

```suggestion
- Configure Redis using the same conventions described in [distributed caching](./distributed-caching#redis-url-configuration).
```

Reply:
just the configuration is fine, because the entity cache does not use the in memory cache